rmgpy/cantherm/commonTest.py

```diff
@@ -265,7 +266,20 @@ def testTransitionStateStatmech(self):
         job.includeHinderedRotors = self.useHinderedRotors
         job.applyBondEnergyCorrections = self.useBondCorrections
         job.load()
-
-
+        
+class testStatmech(unittest.TestCase):
+    """
+    Contains unit tests of statmech.py
+    """
+    def setUp(self):
+        cantherm = CanTherm()
+        jobList = cantherm.loadInputFile(os.path.join(os.path.dirname(os.path.abspath(__file__)),r'files/Benzyl/input.py'))
```

Comment:
So when I try to do this I think nosetests starts trying to run the methoxy.py file in the test_data folder or at least it's giving me this error when I run unit tests:
====================================================================
ERROR: Failure: NameError (name 'transitionState' is not defined)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/Cellar/numpy/1.12.0/libexec/nose/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/local/Cellar/numpy/1.12.0/libexec/nose/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/local/Cellar/numpy/1.12.0/libexec/nose/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmgpy/cantherm/test_data/methoxy.py", line 13, in <module>
    transitionState(
NameError: name 'transitionState' is not defined